hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/io/storage/row/TestHoodieRowCreateHandle.java

```diff
@@ -187,7 +187,9 @@ public void testInstantiationFailure(boolean enableMetadataTable) {
 
     try {
       HoodieTable table = HoodieSparkTable.create(cfg, context, metaClient);
-      new HoodieRowCreateHandle(table, cfg, " def", UUID.randomUUID().toString(), "001", RANDOM.nextInt(100000), RANDOM.nextLong(), RANDOM.nextLong(), SparkDatasetTestUtils.STRUCT_TYPE);
+      table.getMetadata();
+      // After #8452, the exception is caused by the creating of metadata in HoodieTable
+      // new HoodieRowCreateHandle(table, cfg, " def", UUID.randomUUID().toString(), "001", RANDOM.nextInt(100000), RANDOM.nextLong(), RANDOM.nextLong(), SparkDatasetTestUtils.STRUCT_TYPE);
```

Comment:
The test logic seems changed, we should fix it.

Reply:
Changes in this place will be reverted, see https://github.com/apache/hudi/pull/9310. Keep it like this for now and repair it when revert?